apps/studio/components/layouts/LogsLayout/LogsLayout.tsx

```diff
@@ -46,7 +46,7 @@ const LogsLayout = ({ title, children }: PropsWithChildren<LogsLayoutProps>) =>
       enabled: showWarehouse,
     }
   )
-  const { data: collections, isLoading: collectionsLoading } = useWarehouseCollectionsQuery(
+  const { data: collections, isFetching: collectionsLoading } = useWarehouseCollectionsQuery(
```

Comment:
any reason why we're changing this? iinw this causes the whole list of warehouse to flicker to the skeleton loader whenever i create a new collection

https://github.com/user-attachments/assets/1a0dfd99-69cd-4794-af1a-7f655479ca7f

Reply:
Also just thought of stress testing long names hahaha - can fix in another PR
![image](https://github.com/user-attachments/assets/ce3ace0f-7df7-4144-8e42-f5d433114c77)

Reply:
> any reason why we're changing this? iinw this causes the whole list of warehouse to flicker to the skeleton loader whenever i create a new collection

when the query is disabled its stuck in isLoading true. I'll find another way to prevent it from showing the loading state on every change, didn't realise that was happening.